src/box/applier.cc

```diff
@@ -1656,9 +1656,8 @@ applier_new(const char *uri)
 	coio_create(&applier->io, -1);
 	ibuf_create(&applier->ibuf, &cord()->slabc, 1024);
 
-	/* uri_parse() sets pointers to applier->source buffer */
 	snprintf(applier->source, sizeof(applier->source), "%s", uri);
-	int rc = uri_parse(&applier->uri, applier->source);
+	int rc = uri_create(&applier->uri, applier->source);
```

Comment:
it's also used in error message in replication.cc. Are you shure that we should drop it?

Reply:
Yes, please do. It's used in replication.cc only because it was the easiest way to log connection name. Add an applier method for reporting connection name instead. I think you could use either `sio_addr_snprint` or `uri_format` there.

Reply:
fixed